spec/pycall/pyobject_wrapper_spec.rb

```diff
@@ -161,7 +161,7 @@ module PyCall
           list = PyCall::List.new([*1..10])
           expect(list[(1..-1).step(2)]).to eq(PyCall::List.new([2, 4, 6, 8, 10]))
           expect(list[(1..-2).step(2)]).to eq(PyCall::List.new([2, 4, 6, 8]))
-          expect(list[(nil..nil).step(-1)]).to eq(PyCall::List.new([*1..10].reverse))
+          expect(list[(10..1).step(-1)]).to eq(PyCall::List.new([*1..10].reverse))
```

Comment:
@annzenkina 
I cannot accept this change because the original expectation is corresponding to Python's `list[::-1]`.
We need to consider how we can emulate Python's `slice(None, None, -1)` in Ruby >= 3.4.

Reply:
How about the followings?

```ruby
(nil..nil).step(-1) # current style
(..nil).step(-1) # shorter version

PyCall::Slice.new(step: -1)
PyCall::Slice.step(-1)
PyCall.slice(step: -1)
PyCall.slice(nil, nil, -1)
PyCall.slice(..nil, -1)
```

Reply:
@mrkn ping